annotations/update_pending_annotations.py

```diff
@@ -0,0 +1,152 @@
+import synapseclient
```

Comment:
You should ideally include a docstring at the beginning of the script to describe its overall purpose.

This will help if someone new takes over the script and/or you return to this script 1-2 years down the line, and need a quick summary/reminder of what it does.

Reply:
Great! I added this.